lrsplines2D/src/LRSplineMBA.C

```diff
@@ -371,14 +393,14 @@ void LRSplineMBA::MBADistAndUpdate_omp(LRSplineSurface *srf, int sgn)
   int kl, kk;
   // const int num_threads = 1;
   // omp_set_num_threads(num_threads);
-#pragma omp parallel default(none) private(kl, kk, el1, el2) shared(nom_denom, tol, dim, el1_vec, el2_vec, umax, vmax, max_num_bsplines, elem_bspline_contributions, kdim, order2, sgn)
+#pragma omp parallel default(none) private(kl, kk, el1, el2) shared(nom_denom, tol, dim, el1_vec, el2_vec, umax, vmax, max_num_bsplines, elem_bspline_contributions, kdim, order2, significant_factor, ssgn)
```

Comment:
Compilation error. Suspecting the "ssgn" variable should be "sgn".

Reply:
Fixed